src/index.ts

```diff
@@ -131,24 +133,27 @@ export class MondayRedisService extends BaseConnector {
     return board.items
   }
 
+  private getChangeKey(itemId: string, title: string) {
```

Comment:
"changesKey" is the name of the *key*. This should probably be "getChangeValue" or better yet "getChangeEntry"